App/Sagas/NodeLifecycle.ts

```diff
@@ -26,12 +28,15 @@ import {
   threads,
   ThreadInfo,
   WalletAccount,
-  version
+  version,
+  registerCafe,
+  cafeSessions,
+  CafeSession
  } from '../NativeModules/Textile'
 import { logNewEvent } from './DeviceLogs'
 import { announcePeer } from './Migration'
 
-const REPO_PATH = RNFS.DocumentDirectoryPath
+export const REPO_PATH = `${RNFS.DocumentDirectoryPath}/textile-go`
```

Comment:
New location for the textile repo. Just to keep things organized and obvious about what files belong to textile-go.